rest-api/src/test/java/io/quarkus/qe/RepositoryResourceTest.java

```diff
@@ -128,8 +147,13 @@ private void whenAddNewRepository() {
                 .body(repository).when().post(PATH);
     }
 
-    private void whenGetRepository() {
-        long entityId = Optional.ofNullable(entity).map(e -> e.id).orElse(AN_ENTITY_ID);
+    private void whenUpdateRepository(Long id) throws RepositoryNotFoundException {
+        response = given().contentType(MediaType.APPLICATION_JSON).accept(MediaType.APPLICATION_JSON).and()
+                .when().put(PATH + "/" + id);
+    }
+
+    private void whenGetRepository(Long defaultEntityId) {
+        long entityId = Optional.ofNullable(entity).map(e -> e.id).orElse(defaultEntityId);
```

Comment:
The defaultEntityId is not needed, the orElse should return the NOT_FOUND_ENTITY_ID field.